packages/shared/providers.yaml

```diff
@@ -3826,6 +3826,29 @@ ramp-sandbox:
         base_url: https://demo-api.ramp.com
     docs: https://docs.nango.dev/integrations/all/ramp
 
+rapidapi:
+    display_name: RapidAPI
+    categories:
+        - dev-tools
+    auth_mode: API_KEY
+    proxy:
+        headers:
+            x-rapidapi-key: ${apiKey}
+            x-rapidapi-host: ${connectionConfig.host}
+        base_url: https://${connectionConfig.host}
```

Comment:
can you change both base_url and host to be something like `https://${connectionConfig.subdomain}.rapidapi.com`?